packages/fiori-elements-writer/test/lrop.test.ts

```diff
@@ -328,7 +329,28 @@ describe(`Fiori Elements template: ${TEST_NAME}`, () => {
                     version: '4'
                 } as unknown as OdataService
             } as FioriElementsApp<LROPSettings>
-        }
+        },
```

Comment:
I am not a fan of testing the new feature like that because I had to scroll through hundreds of lines of snapshot to find one line. Could you please instead test the function  `getAnnotationLibs` directly with a positive and a negative test case.

Reply:
Thank you, i have added some unit tests.

Reply:
Very nice, thank you!